features/step_definitions/event.rb

```diff
@@ -73,3 +73,7 @@
   fill_in "password", with: password
   click_on "Inscription"
 end
+
+Quand("l'utilisateur supprime le point de vue") do 
+  click_on "Supprimer"
```

Comment:
Are you sure this is enough ?
I thought you had a confirmation dialog...

Reply:
![image](https://user-images.githubusercontent.com/57755204/168234967-13e18b76-dfc9-4339-85f1-7667a2a9889a.png)
Indeed it's enough, because the system confirms by default the confirmation dialog

Reply:
Interesting.
Maybe you could follow the advice in the warning, wrapping your code with `accept_confirm`. See https://www.rubydoc.info/gems/capybara/Capybara%2FSession:accept_confirm